core/toolbox/category.ts

```diff
@@ -195,19 +191,15 @@ export class ToolboxCategory
     aria.setState(this.htmlDiv_, aria.State.LEVEL, this.level_ + 1);
 
     this.rowDiv_ = this.createRowContainer_();
-    this.rowDiv_.style.pointerEvents = 'auto';
+    this.rowDiv_.style.pointerEvents = 'none';
```

Comment:
This makes it so the categories are no longer clickable :/ Could you test this manually using `npm run start` and see if `pointerEvents = 'none';` actually needs to be assigned anywhere or if we can remove it?

Reply:
Hey @BeksOmega I tested it in local machine using `npm run start` and it was acting weirdly
and I think the blockyTreeRowContentContainer div is kinda required unless it is breaking 
but we can remove the rowContents_ attributes from the ToolboxCategory class it is redundant...

if you can think of any implementation by which we can remove the blockyTreeRowContentContainer div you can comment I will try it out....

Reply:
Can you try having the `rowDiv_.style.pointerEvents = 'auto'` while the `iconDom_` and the `labelDom_` have it set to `none`? I think that might fix the issue.

Reply:
Hey @BeksOmega sorry for late reply to this... this week was a little busy ....
I have set the iconDom_ and labelDom_ pointerEvents to none and for the rowDiv_ to auto and it is working fine for me... 
please review and suggest for changes if any needed.....